src/components/main/compose/messages/mod.rs

```diff
@@ -343,30 +355,36 @@ pub fn Messages(cx: Scope<Props>) -> Element {
             messages.iter()
                 .zip(next_sender)
                 .zip(prev_sender)
-                .map(|((message, next_sender), prev_sender)| (rg.clone(), message, next_sender, prev_sender))
-                .map(|(mut rg, message, next_sender, prev_sender)| {
+                .map(|((message, next_sender), prev_sender)| {
                     let message_id = message.id();
                     let conversation_id = message.conversation_id();
                     let msg_sender = message.sender();
                     let is_remote = ident.did_key() != msg_sender;
                     let is_last = next_sender.map(|next_sender| *next_sender != msg_sender).unwrap_or(true);
                     let is_first = prev_sender.map(|prev_sender| *prev_sender != msg_sender).unwrap_or(true);
+                    let mut rg = rg.clone();
+
+                    let mut sender_picture = String::new();
+                    if let Some(pfp) = profile_pictures.get(&msg_sender) {
```

Comment:
This could probably be changed to be like the following

```rust
let sender_picture = profile_pictures.get(&msg_sender).and_then(|pbp| pbp.clone()).unwrap_or_default();
```

Reply:
Much cleaner, will change it now